app/utils/common.js

```diff
@@ -7,7 +7,9 @@ import qs from "qs";
 export const getHumanFriendlyDate = (dateString, includeTime = false) => {
   if (!dateString) return null;
   return moment(dateString).format(
-    `MMMM Do, YYYY ${includeTime ? "hh:mm a" : ""}`
+    // make it a bit less human friendly, so it sorts properly
+    `YYYY-MM-DD ${includeTime ? "hh:mm a" : ""}`
+    //`MMMM Do, YYYY ${includeTime ? "hh:mm a" : ""}`
```

Comment:
We should revisit this later. User facing dates should be as human friendly as possible. Ideally we should sort by date objects if that's possible.

Reply:
We could improve maybe but this just admins and the list view is very space constrained.